package/cpp/MmkvHostObject.cpp

```diff
@@ -28,7 +28,21 @@ MmkvHostObject::MmkvHostObject(const facebook::react::MMKVConfig& config) {
   MMKVMode mode = getMMKVMode(config);
 
 #ifdef __APPLE__
-  instance = MMKV::mmkvWithID(config.id, mode, encryptionKeyPtr, pathPtr);
+#include <Foundation/Foundation.h>
```

Comment:
Please move the include up to the other header includes. Don't forget to wrap it in `#ifdef __APPLE__`  again.